src/main/proto/netflix/titus/validator/validator.proto

```diff
@@ -226,3 +230,28 @@ message ImageValidationResponse {
 
     CacheMetadata cacheMetadata = 3;
 }
+
+message NetworkParametersValidationRequest {
```

Comment:
There are already `SecurityGroupValidationRequest` messages in this proto. Why not extend those instead of adding new messages?

Reply:
Looks like it is only missing accountId for our purpose so we should be able to extend it. 
The name will, however, disconnect from what it stands for.

Reply:
There is one difference: unlike `SecurityGroupValidationRequest`, the request in the PR takes `repeated` subnets and security groups. Do you see any other way to achieve the new RPC functionality with the existing Request and RPC?

Reply:
As far as I can tell, `SecurityGroupValidationRequest` isn't actually used anywhere, just halfway wired up some places. If we want to use the existing `SecurityGroupValidationRequest` message we can simply deprecate the existing non-repeated fields and replace them with newer repeated fields. Or we could simply deprecate the messages and gRPC methods using `SecurityGroupValidationRequest` and create all new ones with `NetworkParametersValidationRequest `.

I don't have a strong opinion, just should avoid having two types that are effectively the same thing.

Reply:
I am leaning towards deprecating the SecurityGroup* messages and API in favor of a more general "NetworkAllocation" prefix term.